include/miopen/miopen.h

```diff
@@ -4857,6 +4857,27 @@ miopenStatus_t miopenSetFindOptionResultsOrder(miopenFindOptions_t options,
  */
 miopenStatus_t miopenSetFindOptionWorkspaceLimit(miopenFindOptions_t options, size_t value);
 
+/*! @brief Attaches the preallocated workspace to find options. Allocated by the library by default.
+ *
+ * @param options    Options object to upfate
+ * @param buffer     Specifies the workspace for find call
+ * @param size       Specifies the size of the buffer passed
+ * @return           miopenStatus_t
+ */
+miopenStatus_t
+miopenSetFindOptionPreallocatedWorkspace(miopenFindOptions_t options, void* buffer, size_t size);
+
+/*! @brief Attaches a preallocated tensor to find options. Allocated by the library by default.
```

Comment:
```suggestion
/*! @brief Attaches a preallocated tensor to find options. If not used, buffers are allocated by MIOpen internally by default.
```

Reply:
Maybe also replace "by default" with "(not recommended)"?